conductor-client/src/main/kotlin/com/openlattice/data/storage/BinaryDataWithMetadata.kt

```diff
@@ -0,0 +1,67 @@
+package com.openlattice.data.storage
+
+import com.google.common.base.Preconditions
+import java.util.*
+
+/**
+ *
+ * @author Matthew Tamayo-Rios &lt;matthew@openlattice.com&gt;
+ */
+data class BinaryDataWithMetadata(
+        val contentType: String,
+        val data: ByteArray,
```

Comment:
This is a keyword, gotta ` `` ` it

Reply:
I don't think it is, somehow (although the coloring *is* weird in github)